### PR TITLE
:seedling: Bump kind to v0.19.0

### PR DESCRIPTION
**Pull request by killianmuldoon**
Bump kind to v0.19.0.
Release notes: https://github.com/kubernetes-sigs/kind/releases/tag/v0.19.0

/area dependency

### CODE REVIEW

**Comment by killianmuldoon** [comment]
This release might have breaking changes for existing images - there's a note to use the exact sha for images. Going to do a full test run to see if our current images are still working.

/test pull-cluster-api-e2e-full-main

**Comment by sbueringer** [comment]
> This release might have breaking changes for existing images - there's a note to use the exact sha for images. Going to do a full test run to see if our current images are still working.
> 
> /test pull-cluster-api-e2e-full-main

Yup. Let's definitely see how it goes. But the note is pretty much always there and we didn't have problems in the past.

**Comment by killianmuldoon** [comment]
Just wanted to see given they've done some biggish image changes in this PR. Will update the PR to bump images anyway once / if e2e-full passes.

File: docs/book/src/developer/tilt.md [hunk 1]
```diff
@@ -325,7 +325,7 @@ Custom values for variable substitutions can be set using `kustomize_substitutio
 ```yaml
 kustomize_substitutions:
```
Comment (sbueringer) [inline docs/book/src/developer/tilt.md:326]: @killianmuldoon Can we please also:
* Update `v1.4-to-v1.5.md`
* Update the min kind version in tilt.md
* Update the min kind version in quick-start.md

Regarding the docs, I don't want to keep the old kind version there as we're not testing it

Comment (killianmuldoon) [reply docs/book/src/developer/tilt.md:326]: https://github.com/kubernetes-sigs/cluster-api/pull/8699

**Pull request merged**